test/e2e/apps/statefulset.go

```diff
@@ -95,11 +96,20 @@ var _ = SIGDescribe("StatefulSet", func() {
 		headlessSvcName := "test"
 		var statefulPodMounts, podMounts []v1.VolumeMount
 		var ss *appsv1.StatefulSet
+		var scName string
 
 		ginkgo.BeforeEach(func() {
-			statefulPodMounts = []v1.VolumeMount{{Name: "datadir", MountPath: "/data/"}}
+			if framework.NodeOSDistroIs("windows") {
+				scName, _ = e2epv.GetWindowsStorageClassName(c)
```

Comment:
why do we need to get an OS specific storage class? why can't we use the default?

Reply:
Because currently default storage class has no fsType set. And in-tree GCE PD plugin will default the fsType to ext4 which will not work on Windows.